src/components/App.js

```diff
@@ -1,17 +1,132 @@
 import React from 'react';
+import Search from './Search.js';
+import SearchResults from './SearchResults.js'
+import Team from './Team.js'
+import Game from './Game.js'
 
 import '../styles/components/app.scss';
 
 class App extends React.Component {
 
   constructor(){
     super();
+
+    this.state = {
+      pokemonList: [],
+      searchQuery: "",
+      searchList: [],
+      searchListDetails: [],
+      team: [],
+      opponent: [],
+      play: false
+    }
+
+    this.handleSearchSubmit = this.handleSearchSubmit.bind(this);
+    this.handleSearchChange = this.handleSearchChange.bind(this);
+    this.addPokemonToTeam = this.addPokemonToTeam.bind(this);
+    this.removePokemonFromTeam = this.removePokemonFromTeam.bind(this);
+    this.handleClick = this.handleClick.bind(this);
+    this.generateComputerCard = this.generateComputerCard.bind(this);
   }
 
+componentDidMount(){
+  fetch(`https://pokeapi.co/api/v2/pokemon/`)
+  .then(response => response.json())
+  .then (body => {
+    console.log(body.results)
+    this.setState({
+      pokemonList: body.results
+    })
+  })
+}
+
+
+handleSearchSubmit(){
+  this.setState({
```

Comment:
It would be better to pre-calculate the `searchList`. That way you can set it in state and use it the fetch after without needing the use the callback in setState. Keeps code simpler